gatsby-node.js

```diff
@@ -48,7 +48,7 @@ exports.onCreateNode = async ({ node, getNode, getNodesByType, actions, createNo
     }
     return;
   }   
-  const basePath = category === 'user' ? `docs/en/userguide` : `en`;
+  const basePath = category === 'user' ? `docs/en/userguide` : thirdparty ? `docs/en` : `en`;
```

Comment:
`thirdparty` is truthy for any source that has content after `docs--5`.
This base path is used to decide what part of the file path to remove.
Without this change, the linkfield docs end up at `/en/5/optional_features/linkfield/docs/en/` instead of `/en/5/optional_features/linkfield/`